files/image_config/warmboot-finalizer/finalize-warmboot.sh

```diff
@@ -101,16 +101,30 @@ function check_list()
     echo ${RET_LIST}
 }
 
+function set_cpufreq_governor() {
+    local -r governor="$1"
+    echo "$governor" | tee /sys/devices/system/cpu/cpu*/cpufreq/scaling_governor 1> /dev/null \
+        && debug "Set CPUFreq scaling governor to $governor" \
+        || debug "Failed to set CPUFreq scaling governor to $governor"
+}
+
+function finalize_common() {
+    # Read default governor from kernel config
+    local -r default_governor=$(cat "/boot/config-$(uname -r)" | grep -o 'CONFIG_CPU_FREQ_DEFAULT_GOV_[^=]*=y')
+    set_cpufreq_governor "$default_governor"
```

Comment:
The utilities change is made for MLNX platforms only. I think it would be best to make this call only for MLNX platforms as well.